compiler/src/iree/compiler/Codegen/LLVMGPU/LLVMGPUPromoteMatmulToFitMMA.cpp

```diff
@@ -78,6 +69,12 @@ class LLVMGPUPromoteMatmulToFitMMAPass final
     MLIRContext *ctx = &getContext();
     auto funcOp = getOperation();
 
+    llvm::StringLiteral scheduleAttrName =
```

Comment:
nit: I think we also have a using declaration for this type in `mlir::`
```suggestion
    StringLiteral scheduleAttrName =
```